raycicmd/config.go

```diff
@@ -38,6 +38,11 @@ type config struct {
 	// HookEnvKeys is the list of environment variable keys to pass into
 	// build jobs from buildkite hooks.
 	HookEnvKeys []string `yaml:"hook_env_keys"`
+
+	// Script to populate the buildkite step tags. Then this script is specified,
+	// a buildkite step is skipped if it is tagged by something that is not returned
+	// by this script.
+	TagFilterScript string `yaml:"tag_filter_script"`
```

Comment:
This is more like `TagFilterCommand` now, as it is being launched with bash..

maybe make it a `TagFilterCommand []string`, and run directly without `bash`? This will avoid introducing an external dependency on bash.